app/channels/discussion_post_list_channel.rb

```diff
@@ -0,0 +1,14 @@
+class DiscussionPostListChannel < ApplicationCable::Channel
+  def self.notify!(discussion, iteration)
+    broadcast_to("#{discussion.id}_#{iteration.id}", {})
```

Comment:
So, We want to use the following params:
- `discussion.uuid`
- `iteration.idx`

I don't want to leak DB ids in v3 at all, because it means people can sequentially scrape/step-through everything trying to look at people's content.